.github/workflows/ci.yml

```diff
@@ -607,7 +601,7 @@ jobs:
   small_runtime:
     needs: setup
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'small_runtime')
-    runs-on: ubuntu-20.04-64core
+    runs-on: ubuntu-20.04
```

Comment:
1m30s -> 3m30s

* before: https://github.com/iree-org/iree/actions/runs/8806894791/job/24172721352
* after: https://github.com/iree-org/iree/actions/runs/8806704450/job/24172132208?pr=17145
  * 1 minute pulling from Docker... might be able to use standard packages on the runner with no Docker